<launch>
  <node name="ydlidar_node"  pkg="ydlidar"  type="ydlidar_node" output="screen" respawn="false" >
    <param name="port"         type="string" value="/dev/ttyUSB0"/>  
    <param name="baudrate"         type="int"    value="921600"/>  
    <param name="frame_id"     type="string" value="laser_frame"/>
    <param name="auto_reconnect"    type="bool"   value="true"/>
    <param name="angle_min"    type="double" value="-60" />
    <param name="angle_max"    type="double" value="0" />
    <param name="range_min"    type="double" value="0.1" />
    <param name="range_max"    type="double" value="1.0" />
    <param name="ignore_array" type="string" value="" />
  </node>
  <node pkg="tf" type="static_transform_publisher" name="base_link_to_laser4"
    args="0.2245 0.0 0.2 0.0 0.0  0.0 /base_footprint /laser_frame 40" />
</launch>
